### PR TITLE
doc: Soften and make clearer the language for `no-class-state`

### PR DESCRIPTION
**Pull request by jdforrester**

### CODE REVIEW

**Comment by jdforrester** [comment]
@brion, continuing the conversation from #78, how's this?

**Comment by brion** [comment]
LGTM, that should at least point folks in the right direction and leaves an out when it's necessary. :D

File: rules/no-class-state.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ module.exports = {
 				if ( utils.isjQuery( context, node.callee ) ) {
 					context.report( {
 						node: node,
-						message: 'Don\'t query the DOM for state information'
+						message: 'Where possible, maintain application state in JS and avoid slow DOM queries'
```
Comment (edg2s) [inline rules/no-class-state.js]: "to avoid slower DOM queries"?

**Comment by edg2s** [comment]
We could use similar language in https://github.com/wikimedia/eslint-plugin-no-jquery/blob/master/rules/no-global-selector.js for #109

**Pull request merged**